### DISCUSSION TITLE
Some suggestions for the color scheme

### DISCUSSION THREAD

**Discussion by palexdev** [aliakbarmostafaei/jetbrains-space-theme#10] [Ideas]
Hello @aliakbarmostafaei, thank you for opening this section allowing us to give some feedback, and also thank you for the quickness of your responses

I wanted to discuss with you about two possible changes, one is an improvement, the other is more for convenience/better readability.
I'll use this image to discuss the changes:
<details>
<summary>Current scheme</summary>
<p><img src="https://github.com/aliakbarmostafaei/jetbrains-space-theme/assets/16880178/52b37bf2-8802-4582-9429-f844feb460ec" alt="image"></p>
</details>

1) I noticed that `TODO `comments (but also other markers such as `FIXME`) appear on the right scroll bar as red marks. As you can see from the image, this conflicts with errors in the code. Unfortunately, it already happened many times to me, to scroll to a red mark because I thought it was an error in my code and it wouldn't compile, and instead it was just a reminder. I believe the mark color should be changed to something that doesn't conflict with errors at least, but possibly with warnings and such too

2) I don't really dislike how reminders are styled now (the text has background color), but as you can see from the image there is not a clear distinction between normal comments and reminders, because the rest of the text is all gray. Normally this is not a big deal, especially because markings on the scroll bar clearly indicate where reminders are. But I can see missing this distinction when fast scrolling across the codebase. I like the associated colors but I would modify the scheme as follows: 
<details>
<summary>Reminders after</summary>
<p><img src="https://github.com/aliakbarmostafaei/jetbrains-space-theme/assets/16880178/ff3b7a67-43d6-4f1a-845b-d19faeeb6a36" alt="image"></p>
(notice the clear distinction between normal comments and reminders)
</details>

**Comment by aliakbarmostafaei** [comment]
Hey @palexdev 

Thanks for the feedback! You're right the error stripes are added by mistake and they will be removed in the future release.

As for the coloring, I believe it is a matter preference, what you described is the default behavior of IntelliJ for TODO remarks, whereas I drew inspiration from VS Code for the current styling (highlighting only the tag) since I use VSCode too for some projects and I wanted consistency between the two and highlighting the whole comment was too distracting for me when working!

The coloring of the tag description is actually taken from the default comments color in your code but you can easily change it to your preference in the Color Scheme settings page (as I believe you have already figured out):

![Screenshot from 2023-10-18 01-48-23](https://github.com/aliakbarmostafaei/jetbrains-space-theme/assets/103068330/d267b134-c77c-4bda-8f4d-872d81ef1089)

You only need to copy the `Foreground` and `Background` color codes from the `Tag` to the `Description`. Hope this helps

**Comment by aliakbarmostafaei** [comment]
Fix is issued and you will soon find the update on JetBrains Plugin Marketplace. Thanks for making 'Space' even better! ;)

**Comment by palexdev** [comment]
Absolutely, I agree on it being a personal preference
Thanks for the fix! Now my code will be error stripes free haha

Edit:
@aliakbarmostafaei received the update, thanks! I also customized it like this: 
![image](https://github.com/aliakbarmostafaei/jetbrains-space-theme/assets/16880178/b6bdb7f5-366e-4a7c-a0f0-de077ee94379)
The tags now have the description color that is the same as the highlight, and I added back the stripes on the scroll bar, but with a color that doesn't conflict with anything else (I believe), super useful to not forget about TODOs
